examples/data-objects/table-document/src/test/table.spec.ts

```diff
@@ -50,6 +50,8 @@ describeLoaderCompat("TableDocument", (getTestObjectProvider) => {
 
     it("Initially empty", async () => {
         await expect([]);
+
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
         assert.throws(() => tableDocument.getCellValue(0, 0));
```

Comment:
Probably just need to wrap these in braces to satisfy the lint rule?

Reply:
Yep, thx!